actions/keep-alive/action.yaml

```diff
@@ -0,0 +1,81 @@
+name: Keep Alive
+description: Action to keep the repository alive. If 55 days have passed since the last commit, it makes an empty push.
+
+inputs:
+  bypass:
+    description: Force the action to run regardless of the last commit date.
+    required: false
+    default: false
+  bot_token:
+    description: GitHub token for the bot account.
+    required: true
+
+runs:
+  using: composite
+  steps:
+    - name: Calculate days since last commit
+      id: commit_date
+      shell: bash
+      run: |
+        LAST_COMMIT_DATE=$(git log -1 --format=%ct)
+        echo "Last commit date: $LAST_COMMIT_DATE"
+
+        CURRENT_DATE=$(date +%s)
+        echo "Current date: $CURRENT_DATE"
+
+        # Calculate how many days have passed since the last commit
+        DIFFERENCE=$(( ($CURRENT_DATE - $LAST_COMMIT_DATE) / 86400 ))
+        echo "Days since last commit: $DIFFERENCE"
+
+        echo "days_since_commit=$DIFFERENCE" >> $GITHUB_OUTPUT
+
+    - name: Configure git user
+      if: steps.commit_date.outputs.days_since_commit >= 55 || inputs.bypass == 'true'
+      id: git_user
+      shell: bash
+      run: |
+        # Set default values for git configuration
+        # Use the official GitHub Actions bot account as default
```

Comment:
question: what's the "official github actions bot account"?" can we add a link to docs?